packages/truffle-db/src/loaders/artifacts/index.ts

```diff
@@ -433,7 +459,9 @@ export class ArtifactsLoader {
               configNetworks.push({
                 contract: contractName,
                 id: id,
-                address: filteredNetwork[0][1]["address"]
+                address: filteredNetwork[0][1]["address"],
```

Comment:
btw you can do `.find(...)` instead of `.filter(...)[0]` if you want to just filter for a single item. should clean up this code a bunch! (thanks to Harry for telling me about this :)